### ISSUE TITLE
Load specific Bootstrap JS modules

### ISSUE THREAD

**Issue by allcentury** [ruby-bench/ruby-bench-web#8]
Hi Alan & Team,

I see this [trello card](https://trello.com/c/PYgUsEbz/15-don-t-load-all-bootstrappy-stuff) related to the asset pipeline and I'm happy to jump in here.  That said I'm not sure what the plans are for bootstrap's JS within the site so I'm not sure where to start.

ANthony

**Comment by tgxworld** [comment]
Hi @allcentury, thank you for taking interest in the project. :smile: 

I created the card sometime back to remove the all JavaScript modules provided by bootstrap that the project was not using. I ended up just [removing all of it](https://github.com/ruby-bench/ruby-bench-web/commit/d56fefde9edb2f2963102c6d71b5f21647f90ab7) since we were using none if it :sweat_smile: I should have updated the trello card. Sorry about that!
